VideoApp/VideoApp/Stores/Chat/ChatStore.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  Copyright (C) 2021 Twilio, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import TwilioConversationsClient
+
+protocol ChatStoreWriting: AnyObject {
+    var connectionState: ChatConnectionState { get }
+    var messages: [TCHMessage] { get }
```

Comment:
In another ticket I will likely abstract `TCHMessage` here so that I am not exposing and SDK type but also to handle decoding of things for UI like file download messages.